infra/modules/codepipeline/buildspec/deploy-docker-image.yml

```diff
@@ -0,0 +1,18 @@
+version: 0.1
+
+phases:
+  pre_build:
+    commands:
+      - echo Logging in to ECR...
+      - aws ecr get-login-password --region $DEFAULT_REGION | docker login --username AWS --password-stdin $ACCOUNT_ID.dkr.ecr.$DEFAULT_REGION.amazonaws.com
```

Comment:
Do you need to login? Can you just use a role given this is in codebuild?

Reply:
To be honest, I don't know, but it's what the [AWS example](https://docs.aws.amazon.com/codebuild/latest/userguide/sample-docker.html) does.

Reply:
I don't know codebuild etc enough to suggest otherwise, it works I guess